op-challenger2/game/fault/solver/solver.go

```diff
@@ -17,13 +17,15 @@ var (
 type claimSolver struct {
 	trace     types.TraceAccessor
 	gameDepth types.Depth
+	daType    types.DAType
```

Comment:
Looks like `daType` is not used anywhere?

Reply:
currently, it's not used. It's predefined for later usage by Bill and Po